### PR TITLE
Code-rewiew-practice-Week2-day1-activity

### PR DESCRIPTION
**Pull request by fernando-silvabr66**
This code-review activity is part of the Week 2 - Day 1 activities and it is designated to practice the code-review process and activities.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <div class="how-it-works-video">
+        <iframe src="https://www.youtube.com/watch?v=y8Yv4pnO7qc&feature=youtu.be" frameborder="0" id="video">
+
+        </iframe>
+      </div>
+    </div>  
+
+    <div id="pricing">
+      
+    </div>
```
Comment (fernando-silvabr66) [inline index.html:79]: It seems that you left unused code in your project. Please, consider that leaving unused code in your files may make it difficult to understand and evaluate.

**Review by fernando-silvabr66 — COMMENTED** [review]
Hi!, 
Please, check the linters files/configuration as I couldn´t find them attached to your project. 

Also, please review your commits as it seems there is only one. Wouldn´t your project be better documented if you make the commits using more steps?

File: index.html (hunk 1 above)
Comment (fernando-silvabr66) [inline index.html:79]: It seems that you left unused code in your project 😜 . Please, consider that leaving unused code in your files may make it difficult to understand and evaluate 🎉✨.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
```
Comment (mateo951) [inline index.html:9]: Please use [inline comments feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) like this one to show the context of your requirements and help them identify where to make the necessary changes 😉

Comment (mateo951) [reply index.html:9]: Kindly make at least two inline comments in your project files to meet the requirements 📝🤓

Comment (mateo951) [reply index.html:9]: OPTIONAL
This is just a suggestion so you can apply it if you wish 😁To improve your reviews even further, you could use a couple of emojis and/or gifs in the review. The reason for this is to make the code review more human-friendly 👽🤖

**Review by mateo951 — CHANGES_REQUESTED** [review]
Hi @fernando-silvabr66, 

Good job so far with your review 😎!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+
+
+
+
+
+
+
+
```
Comment (fernando-silvabr66) [inline css/style.css:115]: Hi! Here you could try not leaving unused lines 🛠 within your code. It helps make your code clean and save file space as well 🎨💖.

File: index.html (hunk 2 above)
Comment (fernando-silvabr66) [reply index.html:9]: Hi @mateo951 ... kindly ask you to help m in discovering how to configure my GitHub in order to have the emoji and icons available as I couldn´t see them.

**Comment by fernando-silvabr66** [comment]
Hi @mateo951 I´ve left some inline comments in two files (1 for each), please see if that works. Thanks.

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html:9]: Of course, my pleasure! 🔥😎 Try pressing the Windows logo key + . (period). If you are using other operating systems you could try this [link](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md#flags). Your inline comments are great! Go ahead and ask for a re-review 🙌

**Comment by fernando-silvabr66** [comment]
Thanks for showing me about the emojis... (I´ve added some into the comments). 🙌🙌🙌

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi <STUDENT_MENTION>, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Great job! Implementing changes required by the previous code reviewer 

## Suggestions 
I would suggest wrapping your review with a summary just like this one where you let students know what's the general status of the review 😌 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fernando-silvabr66** [comment]
This one was a very good code review process 👏👏👏 a lot of experience sharing has been experienced. Good job!
Alt Text

**Comment by fernando-silvabr66** [comment]
Closing this activity... well done... cheers! 🥇🥇🥇

**Pull request merged**